examples/ex15p.cpp

```diff
@@ -217,14 +220,31 @@ int main(int argc, char *argv[])
    ParFiniteElementSpace flux_fes(&pmesh, &flux_fec, sdim);
    RT_FECollection smooth_flux_fec(order-1, dim);
    ParFiniteElementSpace smooth_flux_fes(&pmesh, &smooth_flux_fec);
```

Comment:
We can't move the RT_FECollection into the switch statement without leaking, because AFAIK the ownership does not transfer to the ParFiniteElement space. See https://github.com/mfem/mfem/blob/kelly-error-indicator-dev/fem/fespace.hpp#L96-L97